tools/pythonpkg/tests/fast/api/test_read_csv.py

```diff
@@ -470,6 +469,8 @@ def test_read_csv_combined(self, duckdb_cursor):
         assert rel.types == rel2.types
 
     def test_read_csv_names(self):
+        pytest.importorskip("fsspec")
```

Comment:
These should actually work, but the use of `StringIO` here implicitly requires `fsspec`. I'll update the tests to use temporary files so that the fsspec module isn't required.